.github/workflows/pull-request.yml

```
@@ -20,12 +20,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 5
+          fetch-depth: 0
 
       - name: "Determine what was removed"
         id: detect-removals
         run: |
-            deleted=$(git diff --name-only --no-renames --diff-filter=D HEAD^ -- */meta.json)
+            deleted=$(git diff --name-only --no-renames --diff-filter=D ${GITHUB_BASE_REF} -- */meta.json)
```

I guess this needs to become something like this:
```suggestion
            deleted=$(git diff --name-only --no-renames --diff-filter=D origin/${GITHUB_BASE_REF} -- */meta.json)
```
... except that won't work if `$GITHUB_BASE_REF` refers to a tag... *sigh*